addons/controllers/clusterbootstrap_controller.go

```diff
@@ -427,6 +428,13 @@ func (r *ClusterBootstrapReconciler) createOrPatchKappPackageInstall(clusterBoot
 				Name: secretName},
 			},
 		}
+
+		if pkgi.Labels == nil {
+			pkgi.Labels = map[string]string{}
+		}
+		pkgi.Labels[types.ClusterNameLabel] = cluster.Name
+		pkgi.Labels[types.ClusterNamespaceLabel] = cluster.Namespace
```

Comment:
question:  could we combine L431-L436 into L376 ?

Reply:
Changed to add labels in the block of L376